ci/management/commands/generate_reports.py

```diff
@@ -141,14 +152,21 @@ def get_identity(self, ids_client, identity):
         self.identity_cache[identity] = identity_object
         return identity_object
 
+    def get_registrations(self, hub_client, **kwargs):
+        registrations = hub_client.get_registrations(kwargs)
+        cursor = registrations['next']
+        while cursor:
+            for result in registrations['results']:
+                yield result
+            params = parse_cursor_params(cursor)
+            registrations = hub_client.get_registrations(params)
+            cursor = registrations['next']
+        for result in registrations['results']:
+            yield result
```

Comment:
If you get the registrations at the beginning of the loop, and then yield the registrations last, then we won't need this extra step (I think I did this before with a `while True`, and then a `break` if `next` is `None`). But we can look into that when we implement this in the client.